Automattic/wp-calypso
# Command Palette: Replace gridicons with plain SVG

## Proposed Changes

The `Gridicon` component from `@automattic/components` leverages `<use>` to [load the SVG sprite](https://github.com/Automattic/wp-calypso/blob/2839c654a3e624a33d61ec0c7889c50d0e0ce805/packages/components/src/gridicon/index.tsx#L46) and that triggers a CORS block when loading it from a different domain: 

<img width="894" alt="Screenshot 2024-02-29 at 15 59 25" src="https://github.com/Automattic/wp-calypso/assets/1233880/2e521050-a015-4699-b646-01fe2639aaa0">

That causes some icons to be missing in the command palette, so this PR replaces all the Gridicons usages with plain SVGs.

Before | After
--- | ---
<img width="446" alt="Screenshot 2024-02-29 at 15 59 50" src="https://github.com/Automattic/wp-calypso/assets/1233880/ec2e8e07-de17-4b68-97d7-dbe4972ac54c"> | <img width="443" alt="Screenshot 2024-02-29 at 17 41 21" src="https://github.com/Automattic/wp-calypso/assets/1233880/f75aef54-6745-47bd-b681-ae85c71e6571">



## Testing Instructions

- Use the Calypso live link below
- Open the command palette in Calypso
- Select a multi-site command
- The site selector should show up
- Make sure the default "globe" icon for sites without icon is displayed correctly
- Apply these changes to your sandbox: `install-plugin.sh command-palette fix/command-palette-gridicons`
- Sandbox `widgets.wp.com`
- Open the command palette in WP Admin (make sure to disable/clear the cache in your browser, otherwise you might be loading a cached command palette)
- Make sure the "Open reader" command displays an icon correctly
